src/main/java/com/netgrif/application/engine/configuration/UserServiceConfiguration.java

```diff
@@ -15,9 +17,15 @@
 @Configuration
 public class UserServiceConfiguration {
 
+    @Value("${nae.ldap.enabled}")
+    protected boolean ldapEnabled;
+
     @Bean
     @ConditionalOnMissingBean
```

Comment:
For this solution may would be good to think about other solution - some kind of config, that tells, which bean to use, this could be defined with some key.